CHANGES.rst

```diff
@@ -19,6 +19,8 @@ Bug fixes:
 3.0.2 (2018-09-28)
 ------------------
 
+Changes:
+
```

Comment:
Oh, forgot to remove that. I'm not doing a new commit to save the planet from unnecessary computing power but directly fix it before an eventual merge.